### DISCUSSION TITLE
Running OpenVINO on WSL (Windows Subsystem for Linux - Ubuntu 20.04)

### DISCUSSION THREAD

**Discussion by raymondlo84** [openvinotoolkit/openvino_notebooks#79] [Show and tell]
Just tested the OpenVINO Notebooks on the WSL + Ubuntu 20.04! Basically you can follow all steps from 20.04 side (make sure you install the python3-dev build-essential and python3-venv package)! 

Then, once you got through the whole pip install step, you just copy and paste the URL and open that on the browser! Magic.

**Comment by raymondlo84** [comment]
![Capture](https://user-images.githubusercontent.com/1720147/118758256-86ced480-b823-11eb-8209-5732c50f4dd7.PNG)

**Reply by paularamo** [reply]
@raymondlo84 Great discussion. Any way for using webcam on WSL or GPU? We cannot open the camera or use the GPU.

**Reply by raymondlo84** [reply]
webcam most likely you will need to pipe it in like an IP webcam...
